docs/userguides/testing.md

```diff
@@ -187,6 +155,40 @@ def my_contract(Contract):
 
 It has the same interface as the [ChainManager](../methoddocs/managers.html#ape.managers.chain.ChainManager).
 
+## Test Pattern
```

Comment:
should rename this to like `"Example"` or something, as the intent has changed by moving it away from the top of the article. It was supposed to be like "this is what high-level tests should be", but when moved down here, it is more like "this is a complete example".